specification/under_development/Instruction_Set.md

```diff
@@ -0,0 +1,43 @@
+# Instruction Set
+
+A backend that supports quantum computations is composed of multiple processing
+units that include one or more quantum processors as well as classic computing
+resources. A quantum program is a combination of instructions that execute on a
```

Comment:
Many current systems do not provide classical computing resources that are accessible to users. E.g. from the point of view of a user, an IBM quantum processor is just that, a quantum accelerator only.  While there maybe classical computing behind the scenes, the QIR sees it only as a quantum device.  Of course the Qiskit Runtime is different and does consist of both. 
I'm thinking the wording on this should be changed to ", and in some services, instructions that execute on an adjacent classical computer"
... or something that makes it clear that different combinations may exist.

Reply:
I wanted to avoid confusing things with mentioning services anywhere - I think we should be able to have a full specification for how the stack works without any assumptions or explanations regarding what runs where. I hence simply added an "in some cases". Does that address your concern?

Reply:
Yes, that seems fine.